cyclopts/core.py

```diff
@@ -1282,11 +1282,25 @@ def __call__(
                 if backend == "asyncio":
                     import asyncio
 
-                    return asyncio.run(command(*bound.args, **bound.kwargs))
+                    try:
+                        # Check if we're already in an async context
+                        asyncio.get_running_loop()
+                        # We're in an async context, return the coroutine for the caller to await
+                        return command(*bound.args, **bound.kwargs)
```

Comment:
Is there a chance a caller might miss the fact that a coroutine was returned and needs to be awaited? Asking because it's looks like an interesting pattern but I'm not sure what part of the code would have to deal with this return.